### PR TITLE
Adding pipeCwd to schema.

### PR DESCRIPTION
**Pull request by rajkumar42**

### CODE REVIEW

**Comment by rajkumar42** [comment]
@gregg-miskelly @caslan @chuckries @DustinCampbell

File: package.json [hunk 1]
```diff
@@ -298,10 +298,16 @@
...
                 "description": "When present, this tells the debugger to connect to a remote computer using another executable as a pipe that will relay standard input/output between VS Code and the .NET Core debugger backend executable (clrdbg).",
                 "default": {
+                  "pipeCwd": "${workspaceRoot}",
                   "pipeProgram": "enter the fully qualified path for the pipe program name, for example 'c:\\tools\\plink.exe'",
                   "pipeArgs": []
                 },
                 "properties" : {
+                  "pipeCwd": {
+                    "type": "string",
+                    "description": "The fully qualified path where the pipe command to execute.",
```
Comment (caslan) [inline package.json]: The description sentence sounds grammatically incorrect. Maybe something like "The fully qualified path to the working directory for the pipe program"?

**Comment by gregg-miskelly** [comment]
NOTE: This might have been your plan already, but I would suggest also updating the MIEngine/OpenDebugAD7 versions in this PR as well.

**Comment by gregg-miskelly** [comment]
Otherwise LGTM

**Comment by caslan** [comment]
LGTM

**Comment by rajkumar42** [comment]
@gregg-miskelly  yep thats the plan. I need to wait for other CR and builds to complete.

**Comment by chuckries** [comment]
:+1:

**Comment by gregg-miskelly** [comment]
:shipit:

**Pull request merged**